src/lang/std/sketch.ts

```diff
@@ -157,8 +160,12 @@ const commonConstraintInfoHelper = (
   const firstArg = callExp.arguments?.[0]
   const isArr = firstArg.type === 'ArrayExpression'
   if (!isArr && firstArg.type !== 'ObjectExpression') return []
+  const pipeExpressionIndex = pathToNode.findIndex(
+    (x) => x[1] === 'PipeExpression'
+  )
+  const pathToBase = pathToNode.slice(0, pipeExpressionIndex + 2)
```

Comment:
These sort of renames are helpful, thank you

Reply:
I just realized 164 could also be `([node]) => node === 'PipeExpression'`

Reply:
done.